society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -309,7 +324,18 @@
         </choose>
       </else-if>
       <else>
-        <text variable="title" quotes="true" text-case="title"/>
+        <!-- Supports citations of introductions, prefaces, or forewords per SBLHS2 § 6.2.14. -->
+        <choose>
+          <if variable="part-title">
+            <group delimiter=" ">
+              <text variable="part-title" suffix=" to "/>
```

Comment:
I know there's no `to` term, but hardcoding "to" makes the style less portable across locales. I'm not sure what would be the best solution. Maybe leave it to the authors to add "to" to `part-title`? That would still be less than ideal, but at least you could use the appropriate terms. @adam3smith  WDYT?

Reply:
Yes, I think adding a "to" term to the specs would be appropriate. I'll open an issue for that, but this will probably take some time.

Reply:
What's the context here? "to" is the verb form of `recipient`

Reply:
Hmm, that seems a little problematic. If the part title is an actual title and not just "Introduction", you get

"John Doe, This is the title of the part not just a descriptor to A very important book, by Jane Smith ..."

Reply:
Good call. Any way to circumvent that? Maybe this: if it has a real title you'd use item type chapter, title instead of part-title, with the book's title ending up in container-title, or so?

Reply:
Also, this encourages style specific data entry, but IDK if there's a better way...

Reply:
I would enter an Introduction as a `chapter`. That's what's appropriate and what we've advised for years and what, eg, Zotero implements with its labeling of ` chapter` as "Book Section". `part-title` isn't really conceptually appropriate. That's intended for something like "Papers on statistical inference, Part 4: Demonstrations".

This is an example of frankly silly style inconsistencies that are difficult to automate. Why should a chapter be cited treating its title as a title, but an introduction treating its title as something like a descriptor?

I don't have a great solution other than to ignore the inconsistency and just cite an introduction like any other book section. I suppose one could do something like entering "Introduction" as `genre` and having no title. But that's more style-specific data entry and would lead, eg, to it appearing as `[Introduction]` in APA style

Reply:
@bwiernik, I totally agree that, conceptually, this is a needless complication in SBL style. But in this respect, SBL style is simply echoing Chicago (e.g., 17th ed., 14.110). That doesn't make it any less of a needless complication. But it's not a _discipline-specific_ needless complication. :-) Using `part-title` for this kind of situation may some have merit in connection with, e.g., CMS, 17th ed., 1.48, in the sense that it groups a section that's on the same organizational level as others with proper part tiles, even though the "introduction" or "preface" (etc.) might not have a part title stated as such. 

In principle, a better option might be to have a variable like "front-matter" that could be called for this specific situation and, therefore, avoid the problems that you point out very fairly for the use of "part-title" here.

Using `recipient` also seems less than obvious but is doubtless better than hard-coding " to ". I'll be happy to make that swap if there seems to be a workable way to accommodate the front matter type information as such.

Thanks so much for your thoughts and, @denismaier, for your careful review of these possible updates.

Reply:
`recipient` will work in English, but not in German.

Reply:
Yeah, "recipient" isn't appropriate for this case. I'll think some more of how to handle this situation

Reply:
@bwiernik this one is waiting for your input